src/main/java/frc/robot/Constants.java

```diff
@@ -1,10 +1,41 @@
 package frc.robot;
 
+import com.ctre.phoenix.motorcontrol.can.TalonFXConfiguration;
+import edu.wpi.first.math.MatBuilder;
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.Nat;
+import edu.wpi.first.math.numbers.N1;
 
 public final class Constants {
+    public static final double LOOP_PERIOD = 0.02; // loop period. [s]
+    public static final double g = 9.80665; // Gravity acceleration constant. [m/s^2]
+    public static final double UPPER_TARGET_HEIGHT = 2.64; // Height of upper target. [m]
+    public static final double PEBZNER_HEIGHT = 4.8; // Height of pebzner auditorium. [m]
+    public static final int TALON_TIMEOUT = 10; // Waiting period for configurations. [ms]
+    public static final double NOMINAL_VOLTAGE = 12.0; // Nominal voltage. [V]
 
-    public static class ExampleSubsystem {
-        public static final double POWER = 0.5; // [%]
-    }
 
+    public static class Shooter {
+        public static final double TICKS_PER_RADIAN = 2048 / (2 * Math.PI); // Ticks per revolution of the shooter motor. [tick]
```

Comment:
Why do we care about radians? we use rps in order to measure the speed of the shooter